libs/store/src/lib/infrastructure/persistence/store/entity-store-server/entity-store-builder.ts

```diff
@@ -0,0 +1,116 @@
+import { Injectable } from '@angular/core';
+import { EntityCollectionServiceFactory } from '@ngrx/data';
+
+import { FetchPolicy, CachePolicy, Type } from '../../../../domain/public_api';
+import { DefaultEntityStore, EntityStore, EntityStoreOptions } from './entity-store';
+import { EntityMetaOptionsService } from '../../entity-options.service';
+
+//#region Interfaces
+
+/**
+ * Entity Store Builder interface
+ */
+export interface EntityStoreBuilder<T> {
+    /**
+     * Apply Cache Policy
+     * @param policy CachePolicy settings
+     */
+    useCachePolicy(policy: CachePolicy): this;
+    /**
+     * Apply Fetch Policy
+     * @param policy FetchPolicy settings
+     */
+    useFetchPolicy(policy: FetchPolicy): this;
+    /**
+     * Created store will not be persisted
+     */
+    isTransient(): this;
```

Comment:
Let's remove this `isTransient` as we dont have use for it yet. The idea behind this, was that I configure EntityStore Server and tell this EntityStore is Transient. Meaning it does not store anything, its not cached. etc.. 

this was related to one of our discussion we had. EntityStore<XXX> will maintain some local map /cache and it will be aware of what entity you currently retrieve using `.getBy` . But I am not sure about it now..

Reply:
changed